packages/components/src/components/tooltip/tooltip.js

```diff
@@ -151,71 +189,176 @@ class Tooltip extends mixin(
     this.tooltip.changeState(
       state,
       Object.assign(detail, { delegatorNode: this.element }),
-      callback
+      () => {
+        // @questions
+        // -- What is the expected behavior if the user clicks on the trigger when the tooltip is already open
+        // -- What's the focus state look like when there is only rich text within a tooltip (no buttons/links/etc)
+        // -- There is a "flash" where the next element (another trigger in the demo) is focused before focus is
+        //      moved to the tooltip. I'm assuming this is due to the debounce. Now that it's triggered on "click"
+        //      for keyboard users do we still want the debounce?
+        const isHidden = state !== 'shown';
+        this.tooltip.element.setAttribute('aria-hidden', isHidden.toString());
```

Comment:
Can we implement the logging setting/unserting `aria-expanded` in floating menu code? Similar logic can be found at: https://github.com/carbon-design-system/carbon/blob/v10.3.0/packages/components/src/components/floating-menu/floating-menu.js#L267-L270